spec/system/prior_authority/view_applications_spec.rb

```diff
@@ -36,16 +29,108 @@
         ]
       )
     )
-    create(:assignment,
-           user: caseworker,
-           submission: application)
+  end
+
+  before do
+    sign_in caseworker
+    visit '/'
+    click_on 'Accept analytics cookies'
     visit prior_authority_root_path
-    click_on 'Start now'
-    expect(page).to have_content 'LAA-1234'
-    click_on 'LAA-1234'
-    expect(page).to have_current_path prior_authority_application_path(application)
-    expect(page).to have_content 'Requested: £80.50'
-    click_on 'test.pdf'
-    expect(page).to have_current_path(%r{/123123123\?response-content-disposition=attachment%3B%20filename%3Dtest\.pdf})
+  end
+
+  context 'when an application is already assigned to me' do
+    before do
+      create(:assignment, user: caseworker, submission: application)
+      click_on 'Start now'
+    end
+
+    it 'shows me that application in the list' do
+      expect(page).to have_content 'LAA-1234'
+    end
+
+    context 'when I click through to the application details' do
+      before { click_on 'LAA-1234' }
+
+      it 'shows summary details' do
+        expect(page).to have_current_path prior_authority_application_path(application)
+        expect(page).to have_content('LAA-1234')
+        expect(page).to have_content(
+          "Requested: £80.50\nService: Pathologist report " \
+          "Representation order date: 02 January 2023\nDate received: 02 March 2023"
+        )
+      end
+
+      it 'shows application details card' do
+        within('.govuk-summary-card', text: 'Application details') do
+          expect(page).to have_content "LAA referenceLAA-1234\nPrison LawNo"
+        end
+      end
+
+      it 'shows primary quote card' do
+        within('#primary-quote.govuk-summary-card') do
+          expect(page).to have_content "Service requiredPathologist report\n" \
```

Comment:
nit. if these `have_contents` fail then you get an error that is harder to decipher - i.e. could not find "this large bit of content' in this 'large bit of content. I prefer the use of composite rspec examples using `.and have_content` for that reason.